molecule/default/molecule.yml

```diff
@@ -4,36 +4,30 @@ dependency:
   name: galaxy
 driver:
   name: docker
-lint:
-  name: yamllint
-  options:
-    config-file: .yamllint.yml
+lint: |
+  set -eux
+  yamllint -c .yamllint.yml .
+  ansible-lint -v
+  flake8
 platforms:
   - name: centos-6
     image: docker.io/linuxsystemroles/centos-6
     privileged: true
   - name: centos-7
-    image: docker.io/linuxsystemroles/centos-7
+    image: centos/systemd
```

Comment:
This is removing the container registry, which is bad. See: https://raesene.github.io/blog/2019/09/25/typosquatting-in-a-multi-registry-world/

Reply:
ok . . . then we should use quay.io instead of docker.io wherever possible

Reply:
. . . or registry.centos.org

Reply:
fixed in https://github.com/linux-system-roles/template/pull/39/commits/ab09c6054c3e41306606d4bac3762cc092bff382